Document the online meetup platform and the CFP process (#3105)
jenkins-infra/jenkins.io: content/events/online-meetup/index.adoc

If you are interested, please contact us using the link:https://groups.google.com/forum/#!forum/jenkins-advocacy-and-outreach-sig[Advocacy and Outreach SIG forum].

What do we offer?

* Hosting of the meetup on the Zoom Webinar platform. We support up to 500 participants, multiple presenters, Q&A, online polls, and other features available in Zoom
* Meetup hosts. We have a number of Jenkins contributors who volunteer to host the meetups if needed
* Promotion through Jenkins resources and social media channels (meetup.com, Twitter, LinkedIn, etc.)
* Publishing of the recordings on the link:https://www.youtube.com/playlist?list=PLN7ajX_VdyaOfwJ-BMZo_JNTIMCMNxlbN[Jenkins YouTube channel]
* Anonymous feedback forms for organizers and speakers

== Resources

* link:https://www.meetup.com/Jenkins-online-meetup/[Jenkins Online Meetup page on meetup.com] - You can find the upcoming events there
* link:https://www.youtube.com/playlist?list=PLN7ajX_VdyaOfwJ-BMZo_JNTIMCMNxlbN[Online Meetup recordings]
* link:https://github.com/jenkinsci/jep/tree/master/jep/13[JEP-13: Jenkins YouTube Channel Policy for Contributors]

> * link:/conduct[Jenkins Code of Conduct] - All events on the Jenkins meetup must adhere the project's code of conduct

Reviewer: ```suggestion
* link:/conduct[Jenkins Code of Conduct] - All events on the Jenkins meetup must adhere to the project's code of conduct
```

> * link:/projects/jam[Local Jenkins meetups] - For those who want to local events

Reviewer: ```suggestion
* link:/projects/jam[Local Jenkins meetups] - For those who want to organize local events
```

== Sponsors

Our meetup.com page and Zoom account are sponsored by the link:https://cd.foundation/[Continuous Delivery Foundation (CDF)].

> We thank the CDF and all foundation members for making this online meetup possible.

Reviewer: ```suggestion
We thank the CDF and all foundation members for making our online meetups possible.
```
